.github/workflows/examles.yml

```diff
@@ -0,0 +1,48 @@
+name: Examples Tests
```

Comment:
file name typo. `examles.yml` to `examples.yml`

Reply:
thanks